data/assets.go

```diff
@@ -0,0 +1,20 @@
+// +build !release
+//go:generate go run assets_generate.go
+
+package data
+
+import (
+	"net/http"
+	"os"
+)
+
+// Assets contains project assets.
+var Assets http.FileSystem
+
+func init() {
+	dir := os.Getenv("OPENSHIFT_INSTALL_DATA")
```

Comment:
can we use GOPATH to set reasonable default?

Reply:
> can we use GOPATH to set reasonable default?

Maybe?  But that will break in Go 1.11 when we move to modules and toss out `GOPATH` ;).  Maybe export a default in your `~/.bashrc` or whatever?  Or compile it in?